wger/core/tests/test_endpoints.py

```diff
@@ -0,0 +1,23 @@
+import json
+from django.urls import reverse
+from rest_framework import status
+from wger.core.tests.base_testcase import WorkoutManagerTestCase
+from django.contrib.auth.models import User
+
+
+class UserEndpointTests(WorkoutManagerTestCase):
+    def test_adds_user(self):
+        """
+Ensure we can create a new User object
+        """
+        url = reverse('core:user:add')
+        data = {'name': 'Boniface'}
+        response = self.client.post(url, data, formart='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(User.objects.count(), 1)
+        self.assertEqual(User.objects.get().name, 'Bomiface')
```

Comment:
This assertion should fail seeing that the user created has 'Boniface' has his/her name.